### PR TITLE
V00 23 xx

### PR DESCRIPTION
**Pull request by gartung**
Added a switch to do a build of gcc which is needed to build rpm on OSX10.10 because clang does not allow nested function.

### CODE REVIEW

**Comment by gartung** [comment]
This will build rpm 4.8

File: cmsBuild [hunk 1]
```diff
@@ -35,6 +35,26 @@ DEFAULT_SPEC_REPOSITORY_MODULE="CMSDIST"
 removeInitialSpace = re.compile ("^[ ]*", re.M)
 cmsBuildFilename = abspath(__file__)
 
+def trace_calls(frame, event, arg):
```
Comment (ktf) [inline cmsBuild]: What is this for?

Comment (gartung) [inline cmsBuild]: I used it while I was debugging why the script seemed to be stuck.

**Comment by gartung** [comment]
Merged in changes to cmsBuild from V00-22-XX

**Pull request merged**